py/test/test_endpointer.py

```diff
@@ -4,7 +4,6 @@
 import unittest
 
 import numpy as np
-
```

Comment:
This is a problem: when I run `pre-commit run --all-files` on my machine, or isort directly, this change gets undone.

The problem is that soundswallower is an internal module, and therefore there's a space between third party import (numpy) and internal imports, but pre-commit-action is not recognizing that here, it's lumping soundswallower with the third party imports.

Maybe a `pip install -e .` is required first?

Reply:
I suppose for just running isort, it's silly to install everything.
I was going to vote against known-first-party because I think isort ought to be able to figure it out, but for performance reasons I have changed my mind.